src/components/Math/ShowMathModal.vue

```diff
@@ -4,6 +4,10 @@
 -->
 
 <template>
+	<!-- TRANSLATORS LaTeX formula, LaTeX is a system to display complex maths and scientific notation/formulae -->
+	<!-- TRANSLATORS There are two ways to show these formulae, Inline, or Display -->
+	<!-- TRANSLATORS Inline means it appears with other content, eg "something [the formula] something" -->
+	<!-- TRANSLATORS This is the title of a modal dialog to edit a LaTeX formula -->
```

Comment:
I don't think this works.
The translation strings need to be moved to computed or other direct js code in that case